pkgs/servers/home-assistant/custom-components/samsungtv-smart/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub, buildHomeAssistantComponent, aiofiles, casttube
+, websocket-client, wakeonlan }:
+
+buildHomeAssistantComponent rec {
+  owner = "ollo69";
+  domain = "samsungtv_smart";
+  version = "0.13.5";
+
+  src = fetchFromGitHub {
+    owner = "ollo69";
+    repo = "ha-samsungtv-smart";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-12wx8HaTEFQZOnxfcuAKFOXPdDpfAl608N1E6gmWqBI=";
+  };
+
+  propagatedBuildInputs = [ aiofiles casttube websocket-client wakeonlan ];
```

```suggestion
  dependencies = [ aiofiles casttube websocket-client wakeonlan ];
```

Newer lingo to specify runtime dependencies.